dtests/src/test/java/io/snappydata/hydra/reactiveCoreUseCase.conf

```diff
@@ -0,0 +1,222 @@
+hydra.Prms-testRequirement = "Test to verify split cluster mode works as expected";
+hydra.Prms-testDescription = "
+This test starts the snappy cluster and spark cluster, then initializes snappyContext.
```

Comment:
Just make sure that  this test is not starting the network server on all the servers in order to test the remote iterator.